pkg/policyviolation/generator.go

```diff
@@ -130,6 +142,10 @@ func (gen *Generator) Run(workers int, stopCh <-chan struct{}) {
 	glog.Info("Start policy violation generator")
 	defer glog.Info("Shutting down policy violation generator")
 
+	if !cache.WaitForCacheSync(stopCh, gen.pvSynced, gen.nspvSynced) {
+		glog.Error("event generator: failed to sync informer cache")
```

Comment:
glog.Error("policy violation controller...")

Reply:
renaming to glog.Error("policy violation generator: failed to sync informer cache"),
As we have sperate controllers for each resources.